site/docs-md/cordova/index.md

```diff
@@ -0,0 +1,46 @@
+
+## Differences between Capacitor and Cordova
+
+Capacitor has a few key differences that require web developers, previously used to Cordova's approach, to change app development workflows.
+
+### Native Project Management
+
+Capacitor considers each platform project a _source asset_ instead of a _build time asset_. That means you'll check your Xcode and Android Studio projects into source control, as well as use those IDEs when necessary for platform-specific configuration and running/testing.
+
+This change in approach has a few implications. First, Capacitor does not use `config.xml` or a similar custom configuration for platform settings. Instead, configuration changes are made by editing the appropriate platform-specific configuration files directly, such as `AndroidManifest.xml` for Android and `Info.plist` for Xcode.
+
+Additionally, Capacitor does not "run on device" or emulate through the command line. Instead, such operations occur through the platform-specific IDE, which provides a faster, more typical experience that follows the standards of app development for that platform. For example, running iOS apps from the command line is not officially supported by Apple anyway, so Xcode is preferred.
+
+While these changes may be concerning to long-time Cordova users, there are worthwhile benefits:
+
+ 1. Updating and modifying native projects through abstracted-away tools such as `config.xml` is error prone and a constant moving target. Becoming more comfortable with platform-specific tooling makes troubleshooting issues that much easier.
+ 2. It's easier to add custom native code that your app needs without having to build a new plugin for it. Additionally, native teams can work alongside web teams on the same project. 
+ 3. Creating more compelling app experiences is now easier since you "own" the native project, such as adding a native UI shell around your web app.
+ 4. More visibility into native project changes and better app maintainability as new mobile operating system versions are released. When breaking changes to Capacitor are introduced or changes are applied to the native project templates, the team will publish step-by-step upgrade instructions to ensure that the update process is as smooth as possible.
+
+### Plugin Management
+
+Capacitor manages plugins in a different way than Cordova. First, Capacitor does not copy plugin source code to your app before building. Instead, all plugins are built as Frameworks (on iOS) and Libraries (on Android) and installed using the leading dependency management tool for each platform (CocoaPods and Gradle/Maven, respectively). Additionally, Capacitor does not modify native source code, so any necessary native project settings must be added manually (for example, permissions in `AndroidManifest.xml`). We think this approach is less error-prone and makes it easier for developers to find help in the community for each specific platform.
```

Comment:
In Android, the plugins have their own `AndroidManifest.xml` and whatever they write there is merged into the app's `AndroidManifest.xml`, so plugins can really set permissions if needed. Then Android Studio (not us) merges all the modules `AndroidManifest.xml` into one.

Reply:
Hmm maybe that would fit better here?  https://capacitor.ionicframework.com/docs/plugins/android

Seems more like an implementation detail.